SqlInstall.ps1

```diff
@@ -0,0 +1,433 @@
+    please revert that operation by removing the right from the local security policy (secpol.msc).
+
+    You will see a screen with the users available on your machine. There you can choose the user that will act as Service Account for your SQL Server Install. This
+    implies that the user has been created beforehand. 
+
+    Note that the dowloaded installation file must be unzipped or an ISO has to be mounted. This will not be executed from this script. This function offers the possibility
+    to execute an autosearch for the installation files. But you can just browse to the correct file if you like.
+
+    .PARAMETER Version will hold the SQL Server version you wish to install. The variable will support autocomplete
+
+    .PARAMETER Edition will hold the different basic editions of SQL Server: Express, Standard, Enterprise and Developer. The variable will support autocomplete
+
+    .PARAMETER Role Will hold the option to install all features with defaults. Version is still mandatory. If no Edition is selected, it will default to Express!
+
+    .PARAMETER StatsAndMl will hold the R and Python choices. The variable will support autocomplete. There will be a check on version; this parameter will revert to NULL if the version is below 2016
+    
+    .PARAMETER Appvolume will hold the volume letter of the application disc. If left empty, it will default to C, unless there is a drive named like App
+
+    .PARAMETER DataVolume will hold the volume letter of the Data disc. If left empty, it will default to C, unless there is a drive named like Data
+
+    .PARAMETER LogVolume will hold the volume letter of the Log disc. If left empty, it will default to C, unless there is a drive named like Log
+
+    .PARAMETER TempVolume will hold the volume letter of the Temp disc. If left empty, it will default to C, unless there is a drive named like Temp
+
+    .PARAMETER BackupVolume will hold the volume letter of the Backup disc. If left empty, it will default to C, unless there is a drive named like Backup
+
+    .PARAMETER PerformVolumeMaintenance will set the policy for grant or deny this right to the SQL Server service account.
+
+    .PARAMETER SaveFile will prompt you for a file location to save the new config file. Otherwise it will only be saved in the PowerShell bin directory.
+
+    .PARAMETER Authentication will prompt you if you want mixed mode authentication or just Windows AD authentication. With Mixed Mode, you will be prompted for the SA password.
+
+    .PARAMETER EnableException
+            By default, when something goes wrong we try to catch it, interpret it and give you a friendly warning message.
+            This avoids overwhelming you with "sea of red" exceptions, but is inconvenient because it basically disables advanced scripting.
+            Using this switch turns this "nice by default" feature off and enables you to catch exceptions with your own try/catch.
+
+    .Inputs
```

Comment:
This can be removed, as well as outputs. Many projects use them, ours generally do not.